exo-go/src/application/initializer_test.go

```diff
@@ -50,56 +46,48 @@ var _ = Describe("Setup", func() {
 		Expect(err).NotTo(HaveOccurred())
 		expectedDockerComposePath := path.Join(appDir, "tmp", "docker-compose.yml")
 		Expect(util.DoesFileExist(expectedDockerComposePath)).To(Equal(true))
-		dockerCompose, err = dockerHelpers.GetDockerCompose(expectedDockerComposePath)
+		dockerCompose, err := docker.GetDockerCompose(expectedDockerComposePath)
 		Expect(err).NotTo(HaveOccurred())
-	})
 
-	It("should create docker-compose.yml version 3", func() {
+		By("set the version to 3")
 		Expect(dockerCompose.Version).To(Equal("3"))
-	})
 
-	It("should list all services and dependencies under 'services'", func() {
+		By("list all services and dependencies under 'services'")
 		for _, serviceName := range allServices {
 			_, exists := dockerCompose.Services[serviceName]
 			Expect(exists).To(Equal(true))
 		}
-	})
 
-	It("should generate an image name for each dependency and external service", func() {
+		By("generate an image name for each dependency and external service")
 		for _, serviceName := range util.JoinStringSlices(internalDependencies, externalDependencies, externalServices) {
 			Expect(len(dockerCompose.Services[serviceName].Image)).ToNot(Equal(0))
 		}
-	})
 
-	It("should generate a container name for each service and dependency", func() {
+		By("should generate a container name for each service and dependency")
```

Comment:
What is the reasoning behind using `By` here? I think it reads strange:
> Describe Initializer by should generate a container name for ...

Why not stick to the good old BDD style:
> Describe Initializer: it generates a container name for ...